src/codeu/chat/util/PlayInfo.java

```diff
@@ -0,0 +1,80 @@
+package codeu.chat.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.*;
+
+/**
+ * Info to store information about an instance of a play
+ */
+public class PlayInfo {
```

Comment:
this would fit better in chat/common than here, because this class represents a data structure that's part of the model of the application, and is (should be) used by both the server and the client.